pq-crypto/s2n_kyber_evp.h

```diff
@@ -0,0 +1,28 @@
+/*
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+#pragma once
+
+#include <openssl/evp.h>
+
+#include "tls/s2n_kem.h"
+
+#if (defined(OPENSSL_IS_AWSLC) && AWSLC_API_VERSION >= 20)
```

Comment:
We should probably hold off on merging this PR until https://github.com/awslabs/aws-lc/pull/691 is merged into AWS-LC since currently AWSLC_API_VERSION is `19`.